src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.test.ts

```diff
@@ -93,6 +102,22 @@ describe('getIndices', () => {
     ).toBe(0);
   });
 
+  it('should work with partial responses', async () => {
```

Comment:
Maybe also add a test for an error response?

Reply:
There's a test for handing errors at the end of this document, do you think its sufficient?

Reply:
I'm talking about an error *response* , not an exception being thrown.

Reply:
@lizozom Addressed based on util code for checking for errors. I'm not sure if there are other error formats I should be testing.